console/console.go

```diff
@@ -105,6 +105,16 @@ func GetConsolePluginCR(consolePort int, serviceNamespace string) *consolev1alph
 					},
 					Authorize: true,
 				},
+				{
+					Type:  consolev1alpha1.ProxyTypeService,
+					Alias: "rosa-prometheus",
+					Service: consolev1alpha1.ConsolePluginProxyServiceConfig{
+						Name:      "prometheus",
+						Namespace: serviceNamespace,
+						Port:      9339,
+					},
+					Authorize: false,
+				},
```

Comment:
can we maybe add it optionally (only if it is ROSA infra) ?

Reply:
that requires us to detect platform and would add unnecessary complications to our code.